htdocs/install/mysql/migration/5.0.0-6.0.0.sql

```diff
@@ -572,7 +572,7 @@ UPDATE llx_bank SET label= '(SupplierInvoicePayment)' WHERE label= 'Règlement f
 UPDATE llx_bank SET label= '(CustomerInvoicePayment)' WHERE label= 'Règlement client';
 UPDATE llx_bank SET label= '(payment_salary)' WHERE label LIKE 'Règlement salaire';
 
-ALTER TABLE llx_mailing_cibles MODIFY COLUMN source_url varchar(255);
+ALTER TABLE llx_mailing_cibles MODIFY COLUMN source_url varchar(1000);
```

Comment:
Do you have example of url > 255 chars ? Is it external url ?
Note that source_url should be a relative url.
It is annoying to increase to higher value than 255 because varchar  255 is not supported on some databases versions.

Reply:
getNomUrl return more than 255 carraters in most of case (and more and more with tooltips info)

Reply:
The field is not dedicated to store the result of getNomUrl. getNomUrl is a full URL + a lot of formatting output enhancement.  This field should get a simple relative url.
Which selector do you use to get formated url ?